auth/src/main/java/org/aerogear/mobile/auth/AuthService.java

```diff
@@ -195,9 +197,10 @@ public void configure(final MobileCore core, final ServiceConfiguration serviceC
      *
      * @param context the current application context
      * @param authServiceConfiguration the configuration of the auth service
+     * @param browserConfiguration the configuration for the browser used during authentication/SSO
      */
-    public void init(final Context context,
-                    final AuthServiceConfiguration authServiceConfiguration) {
+    public void init(final Context context, final AuthServiceConfiguration authServiceConfiguration,
+                    BrowserConfiguration browserConfiguration) {
```

Comment:
is the browser configuration mandatory? I think it can be null and it should be optional. If that is the case, let's keep the existing init method, and add a new one to allow configure the browser

Reply:
Ya its actually not mandatory so I'll refactor this 👍